omahelsinki/settings.py

```diff
@@ -142,6 +141,14 @@
     'npm.finders.NpmFinder'
 ]
 
+# Set custom settings for compressor
+# COMPRESS_OUTPUT_DIR = 'compressed'
+COMPRESS_OFFLINE = True
+
+COMPRESS_PRECOMPILERS = (
+    ('text/x-scss', 'django_libsass.SassCompiler'),
```

Comment:
If you look below, `COMPRESS_PRECOMPILERS` is defined again, so this setting is unnecessary. `django_libsass.SassCompiler` will not work with the tilde style import that our SASS files use, so we're stuck with the `node-sass`